app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -6,6 +6,16 @@ module Resolution
     #   2. The user has only provided one address for their residential and identity document
     #      address or separate residential and identity document addresses
     class ProgressiveProofer
+      # @param [Hash] applicant_pii, keys are symbols and values are strings, confidential user info
+      # @param [Boolean] double_address_verification, flag that indicates if user will have
+      # both state id address and current residential address verified
+      # @param [String] request_ip, IP address for request
+      # @param [Boolean] should_proof_state_id, based on state id jurisdiction, indicates if
+      # there should be a state id proofing request made to aamva
```

Comment:
@zachmargolis Correct me if I'm wrong, but I think that line continuation here requires additional indentation. 

```suggestion
      # @param [Boolean] double_address_verification, flag that indicates if user will have
      #   both state id address and current residential address verified
      # @param [String] request_ip, IP address for request
      # @param [Boolean] should_proof_state_id, based on state id jurisdiction, indicates if
      #   there should be a state id proofing request made to aamva
```

Reply:
I wasn't sure, so I checked it out locally and confirmed that the continuing indendation is needed
```
bundle exec yard server --reload -- app/services/proofing/resolution/progressive_proofer.rb
```

But it also turns out that the commas after the property names are messing things up, and a few of these were missing  the `@` character

It would be nice to fix this, but we do not enforce correct YARD documentation on our codebase, only on a few files that we explicitly generate documentation from (analytics_events.rb and tracker_events.rb), so I wouldn't block a merge based on that